starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,40 @@
-// Names and Input
+var hacker1;
+hacker1 = "claudia";
+console.log("the driver's name is" + " " + hacker1 );
+var hacker2;
+hacker2 = "summer";
```

Comment:
in order to ask the user to input something you would do  var hacker2 = prompt("What is the navigator's name?");